### DISCUSSION TITLE
It's me, hi, i want two way sync

### DISCUSSION THREAD

**Discussion by J-Piza** [zemmyang/Habitica-GoogleTasksSync#7] [Ideas]
First off, amazing work, love this snippet.

You mentined in README that if anyone actually wants two way sync to let you know, this is me, doing that. 

I've got Reclaim.ai hooked up to my google tasks but rarely use it because my dailies and tasks are all in habitica (and I like it that way), if, hoewever, a task I created in Habitica automatically created a task in GTasks, this would trigger reclaim.ai to find a spot for that task in my calendar.

This is the future I want to live in, so yes, two way sync pretty please.

**Comment by zemmyang** [comment]
neat!

i'll look into it, but no promises yet :)

**Reply by J-Piza** [reply]
Great! thank you!

**Reply by parthatom** [reply]
Me too! Two-way sync, please. I'd love to help out as well, so let me know if you'd need help
